src/EDI/Encoder.php

```diff
@@ -1,25 +1,58 @@
 <?php
+
 /**
  * EDIFACT Messages Encoder
  * (c)2016 Stefano Sabatini
  */
 
 namespace EDI;
 
-class Encoder
-{
-    private $output='';
+class Encoder {
+
+    private $output = '';
+    private $UNAActive = false; // disable by default to preserve backward compatibility 
+    private $array = array();
+
+    /**
+     * @var string : component separator character (default :)
+     */
+    private $sep_comp;
```

Comment:
i made it symmetrical with Parser, that's were those $sep_xxx are comming from, but i agree that s not consistent.